ui/raidboss/data/05-shb/raid/e11s.js

```diff
@@ -614,6 +615,14 @@ export default {
           cn: '八方 -> 分摊 -> 直线 -> 击退 -> 集合',
           ko: '8산개 -> 파트너 쉐어뎀 -> 직선 장판 -> 넉백 -> 모이기',
         },
+        ttsText: {
+          en: 'Cycle of Faith: Fire',
```

Comment:
Do you think it'd be worth being even briefer here and just saying "Fire Cycle"? (and the equivalent in other languages)?

Reply:
I don't know how English speakers call that skill name, in usual, so I just tried to copy the skill name from client data.
If English players call that skill just "Cycle" it's good to change to short.

Also, I understand English but I don't any other. So, I don't know how to short them.